modules/graphics/mrblib/01-origin_coords.rb

```diff
@@ -0,0 +1,23 @@
+module Moon
+  module OriginCoords
+    extend TypedAttributes
+
+    attribute :origin,  Vector2
+
+    def ox
```

Comment:
Let's get rid of these helpers completely I think. Makes much more sense, since we won't be providing helpers for position or any of the other attribs either.